Examples/IcmpFileTransfer/IcmpFileTransfer-pitcher.cpp

```diff
@@ -36,12 +36,15 @@ void usleep(__int64 usec)
 	LARGE_INTEGER ft;
 
 	ft.QuadPart = -(10 * usec);  // Convert to 100 nanosecond interval, negative value indicates relative time
-
+	                             // NULL is used instead of nullptr for Windows APIs. Check
+	                             // https://devblogs.microsoft.com/oldnewthing/20180307-00/?p=98175
```

Comment:
@DeepakReddy1999 I think the alignment of the comment is wrong, no?

Reply:
Yeah ,I just changed it now